src/main/java/br/com/annuum/capsicum/api/config/GeometryFactoryConfig.java

```diff
@@ -0,0 +1,17 @@
+package br.com.annuum.capsicum.api.config;
+
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.geom.PrecisionModel;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class GeometryFactoryConfig {
+
+    private static final int SRID = 4326;
```

Comment:
Eu não curto comentário em código, mas acho que esse SRID carece de uma explicaçãozinha. hehehehe

Reply:
hahahaha, beleza! **Spatial Resource Identifier** (SRID). O valor 432 indica o padrão **World Geodetic System of 1984** (WGS 84)